WORKSPACE

```diff
@@ -2304,20 +2319,27 @@ rpm(
 )
 
 rpm(
-    name = "gsettings-desktop-schemas-0__40.0-6.el9.s390x",
```

Comment:
Oh, wondering what brought gsettings rpm to our deps

Reply:
I found it strange to see gsettings there too - its was added here 2 years ago. https://github.com/kubevirt/kubevirt/commit/6fa2fcaf228910c685ed1ad96a94e76beafcebfd